package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "json-schema-himarc",
-  "version": "2.4.1",
+  "version": "2.4.2",
```

```suggestion
  "version": "2.5.0",
```

j'ai l'impression qu'on devrait faire une montée de version mineure pour ça